### ISSUE TITLE
Bigger size, due to some references that are not needed.

### ISSUE THREAD

**Issue by chebedzic** [locus84/Locus-Bundle-System#21]
Hi,

I really like this repo, but i have an issue with the fact that when i create a bundle it draws multiple assets from my project, including for example fonts, which are huge for our game due to the fact that we support Hindu, Chinese,  Korean and Japanese markets.

My question is if there is a way to not include some dependancies in the asset since they will be included in the build?

**Comment by chebedzic** [comment]
I forgot to mention there will be duplicate fonts in the game by the end of it. because one will be packed due to references from starting scenes, and the other one will be from bundles

**Comment by locus84** [comment]
> I forgot to mention there will be duplicate fonts in the game by the end of it. because one will be packed due to references from starting scenes, and the other one will be from bundles

You're right, what I've done is just make entire game (specially UIs) into assetbundles, except initial logo scene which does not include any Text element.
And then make all other UIs are from local asset bundles. (remote bundles, if needed)
In that way, I have only one dynamic font reference throughout gameplay.

If you're using TextMeshPro, there's a function named TMP_Text.OnFontAssetRequest so you can implement your own load functionality.

**Comment by chebedzic** [comment]
Thanks a lot man i will probably end up doing the same
